components/admin/UserMenu.tsx

```diff
@@ -21,14 +22,15 @@ export function UserMenu(props: {
   data: FragmentType<typeof UserMenuFragment>
 }) {
   const data = useFragment(UserMenuFragment, props.data)
+  const mediaUrl = process.env.NEXT_PUBLIC_MEDIA_URL ?? ''
 
   return (
     <Menu as="div" className="relative">
       <MenuButton className="-m-1.5 flex items-center p-1.5">
         <span className="sr-only">Open user menu</span>
         <img
-          alt=""
-          src="https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"
+          alt={data.currentUser.firstName + data.currentUser.lastName}
```

Comment:
Using the concatenation like this would join the names into a single word.

For example if the `firstName` would be John and `lastName` would be Doe it would joined them to a string `JohnDoe`.

We can use the template string and interpolation (printing variables into a string):

```suggestion
          alt={`${data.currentUser.firstName} ${data.currentUser.lastName}`}
```

This way it would return "John Doe".